packages/compass-e2e-tests/helpers/commands/save-favorite.ts

```diff
@@ -1,13 +1,31 @@
+import { TEST_MULTIPLE_CONNECTIONS } from '../compass';
 import type { CompassBrowser } from '../compass-browser';
 import * as Selectors from '../selectors';
 import { expect } from 'chai';
 
 export async function saveFavorite(
   browser: CompassBrowser,
   favoriteName: string,
-  color: `color${number}`
+  // TODO(COMPASS-8003): make color optional for multiple connections
+  color: `color${number}` | string
```

Comment:
Come to think of it this whole command might also just go away along with single connections because you can now save a favorite via the connection form with setConnectioFormState().